addons/web/controllers/main.py

```diff
@@ -710,7 +710,7 @@ def __init__(self, fields, row_count=0):
         self.fields = fields
 
     def write_group(self, row, column, group_name, group, group_depth=0):
-        group_name = group_name if isinstance(group_name, str) else group_name and group_name[1] or _("Undefined")
+        group_name = group_name if type(group_name) in [str, int] else group_name and group_name[1] or _("Undefined")
```

Comment:
The export feature also crashes if you group by a Boolean field.
I would fix it like:
```suggestion
        group_name = group_name[1] if isinstance(group_name, tuple) and len(group_name) > 1 else group_name or _("Undefined")
```
It's probably more robust